src/Data/API/BaseApiHelper.ts

```diff
@@ -33,26 +35,32 @@ export class BaseApiHelper implements ApiHelper {
 
   private getHeaders(environment?: string): Record<string, string> {
     const { keycloak } = this;
+    const jwt = getCookie("inmanta_user");
+    let Authorization;
+    if (keycloak && keycloak.token) {
+      Authorization = `Bearer ${keycloak.token}`;
+    } else if (jwt) {
+      Authorization = `Bearer ${jwt}`;
+    }
     return {
       ...(environment ? { "X-Inmanta-Tid": environment } : {}),
-      ...(keycloak && keycloak.token
-        ? { Authorization: `Bearer ${keycloak.token}` }
-        : {}),
+      ...(Authorization ? { Authorization } : {}),
     };
   }
 
   private formatError(message: string, response: Response): string {
     let errorMessage = message;
-
-    if (this.keycloak && (response.status === 401 || response.status === 403)) {
+    if (response.status === 401 || response.status === 403) {
```

Comment:
the glitch was caused by Initializer being in wrong place. I can move the logic here, I guess.

Reply:
but this is the closest level you could possibly handle the error, so it would make sense to actually handle the responses here, not in the error-view.